test/e2e/framework/job/wait.go

```diff
@@ -99,22 +101,30 @@ func WaitForJobGone(c clientset.Interface, ns, jobName string, timeout time.Dura
 	})
 }
 
-// CheckForAllJobPodsRunning uses c to check in the Job named jobName in ns is running. If the returned error is not
-// nil the returned bool is true if the Job is running.
-func CheckForAllJobPodsRunning(c clientset.Interface, ns, jobName string, parallelism int32) (bool, error) {
+// EnsureJobPodsRunning uses c to check in the Job named jobName in ns
+// is running, returning an error if the expected parallelism is not
+// satisfied.
+func EnsureJobPodsRunning(c clientset.Interface, ns, jobName string, parallelism int32) error {
 	label := labels.SelectorFromSet(labels.Set(map[string]string{JobSelectorKey: jobName}))
 	options := metav1.ListOptions{LabelSelector: label.String()}
 	pods, err := c.CoreV1().Pods(ns).List(options)
 	if err != nil {
-		return false, err
+		return err
 	}
+	pods := make([]string, 0, parallelism)
```

Comment:
This name clashes with the previous pods and confuses compiler.

Reply:
> This name clashes with the previous pods and confuses compiler.

Fixed in 3e621c70d3f -> 96b04bfeacd, squashing your wking/kubernetes#1.  Thanks :).